src/components/Cards/Card.jsx

```diff
@@ -0,0 +1,52 @@
+import './Card.css';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+
+function Card({ name }) {
+    const [poki, setPoki] = useState([]);
```

Comment:
Als `poki` uiteindelijk een object zal zijn, dan zou ik niet voor een array kiezen als default waarde, maar eerder voor `undefined`
```suggestion
    const [poki, setPoki] = useState();
```